### ISSUE TITLE
Thumbnail are not generating.

### ISSUE THREAD

**Issue by hsab** [gregzaal/Gaffer#77]
The addon is unable to generate thumbnails for HDRIS.

I get the following error in console:
```
'C:\Program' is not recognized as an internal or external command,
operable program or batch file.
```

Thank you in advance

**Comment by gregzaal** [comment]
@hsab Can you test by removing your existing version of Gaffer and installing [this](https://github.com/gregzaal/Gaffer/archive/master.zip) instead?

**Comment by gregzaal** [comment]
@hsab does it work?

**Comment by hsab** [comment]
@gregzaal issue has been fixed.
thank you so much and sorry for the late reply.

**Comment by gregzaal** [comment]
No problem :) I've now released v3.0.2 which includes this fix.
